share/ansible_navigator/markdown/welcome.md

```diff
@@ -5,7 +5,7 @@ Some things you can try from here:
 - `:config`                                               Explore the current Ansible configuration
 - `:collections`                                          Explore installed collections
 - `:doc <plugin>`                                         Show a plugin doc
-- `:r, :run <playbook> -i <inventory> -m interactive`     Run a playbook in interactive mode
+- `:run <playbook> -i <inventory>`                        Run a playbook in interactive mode
```

Comment:
Replace `interactive` with `stdout` 
` Run a playbook in stdout (classic) mode`

Reply:
That's right, I missed that it is an interactive prompt description